packages/stencil/src/components/datacom-scroll-button/datacom-scroll-button.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+
+/**
+ *Scroll button component allows to automatically scroll to specific anchor point when click it.
+ */
+@Component({
+  tag: 'datacom-scroll-button',
+  styleUrl: 'datacom-scroll-button.css',
+  shadow: true,
+})
+export class DatacomScrollButton {
+  @Prop() url: string;
+  @Prop() ctaTitle: string;
+
+  chevronIcon = getSvg('chevron', { class: 'dc-scroll-btn-chevron' });
+
+  render() {
+    return (
+      <Host>
+        <a href={this.url} class="dc-scroll-btn" title={this.ctaTitle}>
```

Comment:
Is this intended to take any kind of url? As that's how it will work currently with how you've developed it. If not, what are examples of what kind of value this href should take?

Reply:
Yup, you are right. It is an old version and still working on it with a different approach.